### PR TITLE
sql/postgres: same inspect, but faster

### PR DESCRIPTION
**Pull request by a8m**

### CODE REVIEW

File: doc/md/ddl/sql_types.md [hunk 1]
```diff
@@ -187,6 +187,7 @@ slug: /ddl/sql-types
             <td>decimal</td>
             <td>
                 <ul>
+                        <li>unsigned (bool)</li>
```
Comment (a8m) [inline doc/md/ddl/sql_types.md:190]: Not part of my PR, but CI is falling.

Comment (masseelch) [reply doc/md/ddl/sql_types.md:190]: Shouldn't we fix it first then and rebase this PR onto that?

Comment (a8m) [reply doc/md/ddl/sql_types.md:190]: I don't mind landing this PR with this failure. It's not a test issue.

Comment (a8m) [reply doc/md/ddl/sql_types.md:190]: Fixed.

**Review by masseelch — APPROVED** [review]

**Pull request merged**